classes/Language.php

```diff
@@ -1109,6 +1112,57 @@ public static function installSfLanguagePack($locale, &$errors = array())
         }
     }
 
+    /**
+     * This method has been introduced in Language out of convenience but, if you can, prefer
+     * using the GenerateMailTemplatesService or MailTemplateGenerator services.
+     *
+     * @param array $langPack
+     * @param array $errors
+     */
+    public static function generateEmailsLanguagePack($langPack, &$errors = array())
+    {
+        $locale = $langPack['locale'];
+        $sfContainer = SymfonyContainer::getInstance();
+        if (null === $sfContainer) {
+            $errors[] = Context::getContext()->getTranslator()->trans(
+                'Cannot generate emails because Symfony container is unavailable.',
```

Comment:
Should we use an article before, _Cannot generate emails because the Symfony container is unavailable._?

Reply:
yes we can